app/assets/javascripts/hidden_values.js

```diff
@@ -28,6 +28,15 @@ function turn_textarea_switch() {
   })
 }
 
+function hidden_value_control(){
+  $(".toggle-hidden-value a").click(function(event){
+    event.preventDefault();
+    var link = $(event.currentTarget);
+    link.find("i").toggleClass("glyphicon-plus").toggleClass("glyphicon-minus");
+    link.parent().toggleClass("unhide");
```

Comment:
we can, but it will require some messing about with the selectors (link.parent().find("span").toggle("hidden"))
the current thing is ugly enough as it is, I don't want to make it even more complicated...